lib/view.dart

```diff
@@ -104,13 +104,24 @@ void clearContentTab() {
   content.children.clear();
 }
 
-void renderAnalysisTabs(List<String> labels) {
-  var wrapper = html.DivElement()..classes = ['row'];
-  var labelCol = html.DivElement()
-    ..classes = ['col-lg-2', 'col-md-3', 'col-sm-12', 'col-xs-12']
-    ..innerText = 'Analyse';
-  var radioCol = html.DivElement()
+html.DivElement getGridRow() {
```

Comment:
nit: how about calling this `generateGridRowElement`? and `generateGridLabelColumnElement` below?

Reply:
Yes this is much clearer, fixed